core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.ts

```diff
@@ -155,6 +156,22 @@ angular.module('oppia').directive('adminMiscTab', [
             });
         };
 
+        ctrl.retrieveExplorationInteractionIds = function() {
+          ctrl.setStatusMessage('Retrieving interactions in exploration ...');
```

Comment:
Should we clear exploration IDs here, too? Otherwise we'll temporarily show the previous lookup's IDs if we look up subsequent explorations.